snaps/learn/tutorials/gas-estimation.md

```diff
@@ -199,10 +257,9 @@ Similarly, you should update the `name`, `version`, `description`, and `reposito
 The `version` field in `snap.manifest.json` inherits the `version` field from `package.json`.
 :::
 
-Lastly, you can update the content of `/packages/site/src/pages/index.tsx`, such as changing the
+You can update the content of `/packages/site/src/pages/index.tsx` by changing the
 name of the method for showing gas fee estimates.
-If you change the method name, make sure to change the method name in `/packages/snap/src/index.ts`
-to match.
+If you change the method name in `/packages/site/src/pages/index.tsx`, ensure you change the method name in `/packages/snap/src/index.ts` to match.
 
 After you have made all necessary changes, you can
 [publish your Snap to npm](../../how-to/publish-a-snap.md#publish-your-snap).
```

Comment:
Update link to `../../how-to/publish-a-snap.md`.

Reply:
Thanks @alexandratran will address in new PR